pulsar-common/src/main/java/org/apache/pulsar/common/util/SecurityUtility.java

```diff
@@ -110,8 +110,8 @@ public static Provider getProvider() {
             return getBCProviderFromClassPath();
         } catch (Exception e) {
             log.warn("Not able to get Bouncy Castle provider for both FIPS and Non-FIPS from class path:", e);
-            throw new RuntimeException(e);
         }
+        return null;
```

Comment:
`Optional` will be better than null.

Reply:
We don't use this TLS provider, the purpose of this is to add a BC provider to the JDK.